src/campaign_server/campaign_server.cpp

```diff
@@ -623,7 +623,7 @@ void server::handle_request_campaign_list(const server::request& req)
 
 			for(const config& j : i.child_range("translation"))
 			{
-				if(j["language"] == lang) {
+				if(j["language"] == lang && j["supported"].to_bool(true)) {
```

Comment:
`supported` should default to `false` in all contexts (especially this one) instead of `true`. This can also be achieved by calling `to_bool()` without an argument.

Reply:
I believe it's made for the backwards compatibility, so if the `supported` attribute is abscent, it will be assumed as true. Newer addons will always have either `supported=yes or no` variable.

Reply:
I think with the sanity check `j["supported"].to_bool(true)` will be redundant, so ok.

Reply:
That's a good point, I forgot that `supported` is going to be entirely absent from campaignd configs initially. In that case it should default to `false` across the board and it should be guaranteed to be present in the config for new/updated uploads.

Reply:
I think it will be defaulted to true on the client. But the server will guarantee it for the new ones as you said.

Reply:
pffft, I meant it should default to `true`, yes. Otherwise what I said doesn't make any sense.

Reply:
Also, regarding the client side: it should also default to `true`, since if the server doesn't send it it means it's an old server. Which means that it will certainly only emit the `translations` tags for translations that have catalogues.